packages/modules/payment/src/models/payment-collection.ts

```diff
@@ -103,12 +103,12 @@ export default class PaymentCollection {
   payment_providers = new Collection<Rel<PaymentProvider>>(this)
 
   @OneToMany(() => PaymentSession, (ps) => ps.payment_collection, {
-    cascade: [Cascade.PERSIST, "soft-remove"] as any,
+    cascade: [Cascade.PERSIST] as any,
```

Comment:
unfortunately had to remove this from the cascade as its causing an ambiguity in the SQL generated as both payment session and payment have the same column called "payment_collection_id"